akka-docs/src/test/scala/docs/stream/TwitterStreamQuickstartDocSpec.scala

```diff
@@ -151,7 +152,9 @@ class TwitterStreamQuickstartDocSpec extends AkkaSpec {
     //#fiddle_code
     tweets
       .filterNot(_.hashtags.contains(akkaTag))
-      .mapConcat(_.hashtags)
+      .map(_.hashtags)
+      .reduce(_ ++ _)
```

Comment:
since this is a doc example, and people may be new to this, how about we add a quick comment next to each line what we're achieving here? We also get scala beginners, so esp the Quickstart should be beginner friendly :)